src/Synapse/Synapse/Commands/ManagementCommands/SqlPool/GetAzureSynapseSqlPoolRestorePoint.cs

```diff
@@ -37,11 +38,21 @@ public class GetAzureSynapseSqlPoolRestorePoint : SynapseManagementCmdletBase
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
+        [Parameter(Mandatory = true, ParameterSetName = GetByNameParameterSet, HelpMessage = HelpMessages.SqlPoolName)]
+        [ResourceNameCompleter(ResourceTypes.SqlPool, nameof(WorkspaceName))]
+        [ValidateNotNullOrEmpty]
+        public string SqlPoolName { get; set; }
+
         [Parameter(ValueFromPipeline = true, ParameterSetName = GetByParentObjectParameterSet,
             Mandatory = true, HelpMessage = HelpMessages.WorkspaceObject)]
         [ValidateNotNull]
         public PSSynapseWorkspace WorkspaceObject { get; set; }
 
+        [Parameter(ValueFromPipeline = true, ParameterSetName = GetByParentObjectParameterSet,
+            Mandatory = true, HelpMessage = HelpMessages.SqlPoolObject)]
+        [ValidateNotNull]
+        public PSSynapseSqlPool SqlPoolObject { get; set; }
+
```

Comment:
No need to add SqlPoolName and SqlPoolObject here. The parameter -Name is sql pool name and -InputObject is sql pool object.
Should add a parameter called -RestorePointCreationDate